opencti-platform/opencti-front/src/private/components/common/stix_relationships/StixRelationshipsList.jsx

```diff
@@ -3390,6 +3390,9 @@ const stixRelationshipsListQuery = graphql`
             ... on Malware {
               name
             }
+            ... on MalwareAnalysis {
+              result_name
+            }
```

Comment:
needs to be duplicated everywhere the name is retrieved on other entities as well. But we should be able to get rid of all 
```
... on X {
   name
}
```
by retrieving the main representative only once on the stixCoreObject 
```
... on StixCoreObject {
   created_at
   representative{
      main
   }
}
```